webapp/website/index.html

```diff
@@ -29,6 +29,7 @@
 	<link rel="icon" type="image/x-icon" href="favicon.ico" />
 	<link rel="stylesheet" type="text/css" href="css/libs.min.css" />
 	<link rel="stylesheet" type="text/css" href="css/app.min.css" />
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
```

Comment:
We should take into account that the user might want to install the app on a computer with no internet or limited internet connection. Thus we should server all  necessary file for the frontend to be used.